src/Plugin.php

```diff
@@ -86,6 +92,46 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes SVG width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
+      return preg_replace('@(?:width|height)=.+?\s+@', '', $matches[0]);
+    }, $content);
+    return $content;
+  }
+
+  /**
+   * Removes SVG width and height attributes when the metadata for the attachment
+   * is received. E.g when calling get_attachment_image() or get_post_thumbnail().
```

Comment:
Yes, you're right. I'll shorten this and drop the example functions as this is clear by the `@implements` doc.